roles/horizon/tasks/main.yml

```diff
@@ -116,3 +117,10 @@
 
 - include: monitoring.yml tags=monitoring,common
   when: monitoring.enabled|default('True')|bool
+
+- name: add horizon customization module
+  template: src=etc/openstack-dashboard/horizon_customization.py
+            dest={{ openstack_source.virtualenv_base }}/horizon/lib/python2.7/site-packages/openstack_dashboard/horizon_customization.py
+            mode=0644
+  notify:
+    - restart apache
```

Comment:
currently looking into creating a python package so we don't path the virtual env